Makefile

```diff
@@ -132,6 +132,22 @@ MIPS_VERSION := -mips2
 # we support Microsoft extensions such as anonymous structs, which the compiler does support but warns for their usage. Suppress the warnings with -woff.
 CFLAGS += -G 0 -non_shared -fullwarn -verbose -Xcpluscomm $(IINC) -nostdinc -Wab,-r4300_mul -woff 649,838,712,516
 
+# The system heap is an area of memory that covers all the VRAM that is not
+# used by compiled code/data.
+# It starts just after the end of compiled code (that is not an overlay) and
```

Comment:
"compiled code" is not wrong but it may be misleading to someone as refering to text only

If it's correct maybe
```suggestion
# It starts just after the end of the code segment
```
Would be clearer?

Reply:
Yours isn't totally correct. The system heap doesn't start after the `code` segment, it starts after the `buffers` segment (which starts after the `code` segment).
I tried to avoid referring to specific segments, but I don't have a good reason to do so either. What do you think to change the comment to refer to the `buffers` segment instead?